src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/finallocalvariable/InputFinalLocalVariableConstructor.java

```diff
@@ -0,0 +1,37 @@
+/*
+FinalLocalVariable
+validateEnhancedForLoopVariable = (default)false
+tokens = PARAMETER_DEF
+
+
+*/
+
+
+package com.puppycrawl.tools.checkstyle.checks.coding.finallocalvariable;
+
+public class InputFinalLocalVariableConstructor {
+
+    InputFinalLocalVariableConstructor(int a) {
+        // violation above 'Variable 'a' should be declared final'
+    }
+
+    InputFinalLocalVariableConstructor(int a, int b) { // 2 violations
```

Comment:
@romani How do we handle this for BDD?

Reply:
Keep it as is, we did not find good way to reflect 2 violations inline.
Please be welcome to suggest something.

Better to avoid this, but fine to have it.

Reply:
@Kevin222004 I assume you have no restriction for this to be on 1 line. Please split between 2 lines, and put the message for each one said line.

Reply:
done